common/decrees/00_decree.txt

```diff
@@ -600,6 +599,8 @@ decree_encourage_resource_industry = {
 		building_group_bg_whaling_throughput_add = 0.2
 		building_group_bg_oil_extraction_throughput_add = 0.2
 		building_group_bg_rubber_throughput_add = 0.2
+		country_capitalists_pol_str_mult = 0.01
+		country_laborers_pol_str_mult = 0.01
```

Comment:
This 1% is too insignificant to add in my opinion, I would suggest making it relevent (like 5%) or removing it. To prevent cluttering.